osu.Game/Overlays/Rankings/SpotlightSelector.cs

```diff
@@ -0,0 +1,165 @@
+
+        private readonly InfoCoulmn startDateColumn;
+        private readonly InfoCoulmn endDateColumn;
+
+        public SpotlightSelector()
+        {
+            RelativeSizeAxes = Axes.X;
+            Height = 100;
+            Children = new Drawable[]
+            {
+                background = new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                },
+                new Container
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Padding = new MarginPadding { Horizontal = UserProfileOverlay.CONTENT_X_MARGIN, Vertical = 10 },
+                    Children = new Drawable[]
+                    {
+                        dropdown = new SpotlightsDropdown
+                        {
+                            Anchor = Anchor.TopCentre,
+                            Origin = Anchor.TopCentre,
+                            RelativeSizeAxes = Axes.X,
+                            Current = SelectedSpotlight,
+                            Depth = -float.MaxValue
+                        },
+                        new FillFlowContainer
+                        {
+                            Anchor = Anchor.BottomRight,
+                            Origin = Anchor.BottomRight,
+                            AutoSizeAxes = Axes.Both,
+                            Direction = FillDirection.Horizontal,
+                            Spacing = new Vector2(15, 0),
+                            Children = new Drawable[]
+                            {
+                                startDateColumn = new InfoCoulmn(@"Start Date"),
+                                endDateColumn = new InfoCoulmn(@"End Date"),
+                            }
+                        }
+                    }
+                },
+                loading = new DimmedLoadingLayer(),
+            };
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            background.Colour = colours.GreySeafoam;
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            SelectedSpotlight.BindValueChanged(onSelectionChanged);
+        }
+
+        public void FetchSpotlights()
+        {
+            loading.Show();
+
+            var request = new GetSpotlightsRequest();
+            request.Success += response =>
+            {
+                dropdown.Items = response.Spotlights;
+                loading.Hide();
+            };
+            api.Queue(request);
+        }
+
+        private void onSelectionChanged(ValueChangedEvent<APISpotlight> spotlight)
+        {
+            startDateColumn.Value = dateToString(spotlight.NewValue.StartDate);
+            endDateColumn.Value = dateToString(spotlight.NewValue.EndDate);
+        }
+
+        private string dateToString(DateTimeOffset date) => $"{date.Year}-{date.Month:D2}-{date.Day:D2}";
```

Comment:
That's a very roundabout way of writing
```suggestion
        private string dateToString(DateTimeOffset date) => date.ToString("yyyy-MM-dd");
```